lib/google_sheets/sheet.rb

```diff
@@ -97,6 +97,19 @@ def self.json_to_csv json
       csv
     end
 
+    ##
+    # Update worksheet values starting from given cell
+    # 
+    # EG: 
+    # ```
+    # sheet.update([['1', 'one'], ['2', 'two']], 'B5')
+    #
+    def update values, starting_cell = 'A1', value_input_option = 'RAW'
```

Comment:
Can we change this to `update!` since it's destructive.  Also make sure `@values` get updated.  Can we also add tests for this in the spec?

Reply:
Hey Sorry Kyle, I completely missed the testing part. Will work on that this weekend and update the PR. Thanks